saleor/core/views.py

```diff
@@ -0,0 +1,11 @@
+import os
+
+from django.template.response import TemplateResponse
+
+
+def home(request):
+    storefront = os.environ.get("STOREFRONT_URL", "")
```

Comment:
If we don't have those set (for ex. someone cloned saleor and running it outside of docker) that will result in broken link in template. Maybe worth adding localhost:3000 as second arg?

Reply:
The link will not be broken, links to the dashboard and storefront won't appear in this situation. If we put there default links and somebody doesn't have running instances of the dash and store it will cause broken links.